### ISSUE TITLE
Switching to HDR mode stops lg working (win11) B7-rc1

### ISSUE THREAD

**Issue by antnn** [gnif/LookingGlass#1140]
Switching to HDR mode stops lg working
AMD RX 560
```
# qemu cmdline
-object '{"qom-type":"memory-backend-file","id":"shmmem-shmem0","mem-path":"/dev/kvmfr0","size":268435456,"share":true}'
-device '{"driver":"ivshmem-plain","id":"shmem0","memdev":"shmmem-shmem0","bus":"pci.16","addr":"0x1"}'
```
![Screenshot 2024-10-27 154121](https://github.com/user-attachments/assets/d30b9a64-840c-4ea7-8234-fd55b778e4f5)

[lg-on-linux.log](https://github.com/user-attachments/files/17533993/lg-on-linux.log)
[looking-glass-host.txt](https://github.com/user-attachments/files/17533972/looking-glass-host.txt)
[looking-glass-host.1.txt](https://github.com/user-attachments/files/17533973/looking-glass-host.1.txt)
[looking-glass-host.2.txt](https://github.com/user-attachments/files/17533974/looking-glass-host.2.txt)
[looking-glass-host.3.txt](https://github.com/user-attachments/files/17533975/looking-glass-host.3.txt)

**Comment by netboy3** [comment]
Please upgrade to Bleeding-Edge (which is very close to release status now).

**Comment by antnn** [comment]
@netboy3 
Still does not work

```
lg client on linux
00:00:00.000 [I]              main.c:1859 | main                           | Looking Glass (4e8201da07)
00:00:00.000 [I]              main.c:1860 | main                           | Locking Method: Atomic
00:00:00.000 [I]           cpuinfo.c:38   | cpuInfo_log                    | CPU Model: 13th Gen Intel(R) Core(TM) i9-13900K
00:00:00.000 [I]           cpuinfo.c:39   | cpuInfo_log                    | CPU: 1 sockets, 24 cores, 32 threads
00:00:00.004 [I]              main.c:1185 | lg_run                         | Using font: /usr/share/fonts/google-noto-vf/NotoSansMono[wght].ttf
00:00:00.005 [I]           ivshmem.c:137  | ivshmemOpenDev                 | KVMFR Device     : /dev/kvmfr0
00:00:00.008 [I]             audio.c:159  | audio_init                     | Using AudioDev: PipeWire
00:00:00.008 [I]                ps.c:245  | purespice_connect              | Connecting to socket 127.0.0.1:5900
00:00:00.014 [I]               rsa.c:178  | rsa_encryptPassword            | Using Nettle
00:00:00.014 [I]                ps.c:268  | purespice_connect              | Connected
00:00:00.014 [I]      channel_main.c:167  | onMessage_mainName             | Guest name: windows11
00:00:00.014 [I]      channel_main.c:183  | onMessage_mainUUID             | Guest UUID: UUID
00:00:00.019 [I]                ps.c:644  | ps_connectChannel              | RECORD channel connected
00:00:00.026 [I]                ps.c:644  | ps_connectChannel              | PLAYBACK channel connected
00:00:00.031 [I]                ps.c:644  | ps_connectChannel              | INPUTS channel connected
00:00:00.031 [I]           channel.c:323  | onMessage_notify               | [notify] keyboard channel is insecure
00:00:00.031 [I]               egl.c:289  | egl_initialize                 | Double buffering is off
00:00:00.031 [I]              main.c:1139 | tryRenderer                    | Using Renderer: EGL
00:00:00.031 [I]           wayland.c:120  | waylandInit                    | Compositor: kwin_wayland_wr
00:00:00.031 [I]           wayland.c:130  | waylandInit                    | Selected  : xdg
00:00:00.038 [I]                gl.c:58   | waylandGetEGLDisplay           | Using eglGetPlatformDisplay
00:00:00.049 [I]               egl.c:856  | egl_renderStartup              | Single buffer mode
00:00:00.051 [I]               egl.c:883  | egl_renderStartup              | EGL     : 1.5
00:00:00.051 [I]               egl.c:884  | egl_renderStartup              | Vendor  : Intel
00:00:00.051 [I]               egl.c:885  | egl_renderStartup              | Renderer: Mesa Intel(R) Graphics (RPL-S)
00:00:00.051 [I]               egl.c:886  | egl_renderStartup              | Version : OpenGL ES 3.2 Mesa 24.1.7
00:00:00.051 [I]               egl.c:887  | egl_renderStartup              | EGL APIs: OpenGL OpenGL_ES 
00:00:00.051 [I]               egl.c:958  | egl_renderStartup              | Debug messages disabled, enable with egl:debug=true
00:00:00.067 [I]           eglutil.c:35   | swapWithDamageInit             | Using EGL_KHR_swap_buffers_with_damage
00:00:00.069 [I]              main.c:1491 | lg_run                         | ================================================================================
00:00:00.069 [I]              main.c:1492 | lg_run                         | The host application seems to not be running
00:00:00.069 [I]              main.c:1493 | lg_run                         | Waiting for the host application to start...
00:00:00.375 [I]              main.c:1498 | lg_run                         | ================================================================================
00:00:00.375 [I]              main.c:1512 | lg_run                         | Check the host log in your guest at %ProgramData%\Looking Glass (host)\looking-glass-host.txt
00:00:00.375 [I]              main.c:1513 | lg_run                         | Continuing to wait...
00:00:01.093 [I]                ps.c:644  | ps_connectChannel              | DISPLAY channel connected
00:00:01.095 [I]              main.c:919  | spice_surfaceCreate            | Create SPICE surface: id: 0, size: 800x600
00:00:01.099 [I]                ps.c:644  | ps_connectChannel              | CURSOR channel connected
```

**Comment by netboy3** [comment]
You are still not running bleeding-edge. Please follow the instructions in the official documentation to upgrade both the host application (the one inside the Windows VM) and the client (on your Linux system) to bleeding-edge. If you still experience issues, please reach out to us and/or the community on either the forum or the Discord server (recommended). As was noted to you when you initially opened this issue, GitHub issues are for reporting verified bug and enhancements and not for general user support issues.

**Comment by antnn** [comment]
@netboy3 solved with `B7-rc1-34-e25492a3`
